docs/release-notes/8.0.41-32.md

```diff
@@ -70,6 +70,20 @@ Find the complete list of bug fixes and changes in the [MySQL 8.0.41 Release Not
 
 * [PXC-4573](https://perconadev.atlassian.net/browse/PXC-4573): The `mysql.pxc.sst.role` lacked the `INNODB_REDO_LOG_ARCHIVE` privilege when the State Snapshot Transfer (SST) required InnoDB redo log archiving.
 
+## Known issues
+
+The Galera Communication System (GCS) protocol has been updated from version 4 to version 5 in recent releases. As a result, if there are plans to upgrade to version 8.4 at a later time, it is important to note that the upgrade will only be supported for versions 8.4.4 and above.
+
+This precaution is necessary due to potential Galera Communication System (GCS) protocol upgrades. If a joining node uses a higher GCS protocol than the existing cluster, the cluster prevents the node from joining.
```

Comment:
should be: `If a joining node uses a lower...`

Higher can join lower. Lower can't join higher.
In other words: protocol upgraded is possible, protocol downgrade is not possible